### ISSUE TITLE
Redmine 5.1.x compatibility

### ISSUE THREAD

**Issue by denisnovak1** [two-pack/redmine_xlsx_format_issue_exporter#100]
Hi, are you planning to add Redmine 5.1.x compatibility for this plugin? Thanks

**Comment by two-pack** [comment]
I haven't checked it yet, but what kind of problem are you having? I'll check it out this weekend, if I have a enough time😅

**Comment by two-pack** [comment]
I've tried a bit, and it seems to be causing an error when exporting the user list, but everything else appears to be working. However, there are failures in the test code, so there might be some other error as well.
I will continue the investigation, but it will take some time.

My env:
```
Environment:
  Redmine version                5.1.1.stable
  Ruby version                   3.2.3-p157 (2024-01-18) [x86_64-linux]
  Rails version                  6.1.7.6
  Environment                    test
  Database adapter               SQLite
  Mailer queue                   ActiveJob::QueueAdapters::InlineAdapter
  Mailer delivery                test
Redmine settings:
  Redmine theme                  Default
SCM:
  Git                            2.34.1
  Filesystem                     
Redmine plugins:
  redmine_xlsx_format_issue_exporter 0.2.0
```

**Comment by two-pack** [comment]
@denisnovak1 
I fixed to ensure compatibility with Redmine 5.1.
I tested on Redmine 5.1.1, 5.07, 4.2.11.
Thank you for contacting :)

NOTE: my env.
```
Environment:
  Redmine version                5.1.1.stable
  Ruby version                   3.2.3-p157 (2024-01-18) [x86_64-linux]
  Rails version                  6.1.7.6
  Environment                    production
  Database adapter               SQLite
  Mailer queue                   ActiveJob::QueueAdapters::AsyncAdapter
  Mailer delivery                smtp
Redmine settings:
  Redmine theme                  Default
SCM:
  Git                            2.34.1
  Filesystem                     
Redmine plugins:
  redmine_xlsx_format_issue_exporter 0.2.1
```

**Comment by denisnovak1** [comment]
Thank you for the information, I will do my tests now on the compatibility :)
